picard/ui/options/dialog.py

```diff
@@ -193,18 +182,17 @@ def disable_page(self, name):
         item = self.page_to_item[name]
         item.setDisabled(True)
 
-    def help(self):
+    @property
+    def help_url(self):
         current_page = self.ui.pages_stack.currentWidget()
         url = current_page.HELP_URL
         # If URL is empty, use the first non empty parent help URL.
         while current_page.PARENT and not url:
             current_page = self.item_to_page[self.page_to_item[current_page.PARENT]]
             url = current_page.HELP_URL
         if not url:
-            url = DOCS_BASE_URL
-        elif url.startswith('/'):
-            url = DOCS_BASE_URL + url
-        webbrowser2.open(url)
+            url = '/config/configuration.html'
```

Comment:
Since the code above, prepend `DOCS_BASE_URL` when url starts with `/`, I think we could remove concatenation in https://github.com/metabrainz/picard/blob/0e30004337a9436b3b57f6b3bd6e8df396fe804a/picard/const/__init__.py#L83-L87

So we could use `PICARD_URLS['doc_options']` here instead of hardcoded `'/config/configuration.html'` or, if we make the change to `open()` I suggested above, just `doc_options`.